debian/cloudstack-usage.postinst

```diff
@@ -40,7 +35,7 @@ case "$1" in
 
         # Replacing key with management server key
         if [ -f "/etc/cloudstack/management/key" ]; then
-            rm -rf /etc/cloudstack/usage/key
+            rm -f /etc/cloudstack/usage/key
             ln -s /etc/cloudstack/management/key /etc/cloudstack/usage/key
```

Comment:
same as above

Reply:
Yes. I'm not sure if the file being removed may not be a symlink, this approach should work as intended.

Reply:
KISS, as far as I'm concerned :)